src/Features/Core/Portable/GenerateEqualsAndGetHashCodeFromMembers/GenerateEqualsAndGetHashCodeFromMembersCodeRefactoringProvider.cs

```diff
@@ -111,7 +111,7 @@ private async Task HandleNonSelectionAsync(CodeRefactoringContext context)
             if (equatableTypeOpt != null)
             {
                 var constructedType = equatableTypeOpt.Construct(containingType);
-                if (!containingType.AllInterfaces.Contains(constructedType))
+                if (!containingType.AllInterfaces.Contains(constructedType) && !containingType.IsRefLikeType)
```

Comment:
Add comment explaining why this is appropriate here.

Reply:
Fixed